src/mlagility/api/script_api.py

```diff
@@ -90,7 +90,7 @@ def benchmark_script(
     analyze_only: bool = False,
     build_only: bool = False,
     resume: bool = False,
-    script_args: str = "",
+    script_args: str = None,
```

Comment:
Wouldn't the following be more appropriate?
```suggestion
    script_args: Optional[str] = None,
```